modules/issue_tracker/ajax/Attachment.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * Issue tracker
+ *
+ * Handles attachments in issue_tracker and
+ * returns data in response for frontend requests.
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  Issue_Tracker
+ * @author   Alizée Wickenheiser <alizee.wickenheiser@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+/**
+ * Issue tracker
+ *
+ * Handles attachments in issue_tracker and
+ * returns data in response for frontend requests.
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  Issue_Tracker
+ * @author   Alizée Wickenheiser <alizee.wickenheiser@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+if ($_SERVER['REQUEST_METHOD'] === 'GET') {
+    if ($_REQUEST['action'] === 'delete') {
```

Comment:
a GET request should never delete something, `GET` is by definition supposed to be idempotent.

Reply:
@driusan I'll try the `DELETE` then